mava/wrappers/environment_loop_wrappers.py

```diff
@@ -98,6 +98,9 @@ def __init__(
             for stat in self._summary_stats:
                 self._running_statistics[f"{stat}_{metric}"] = 0.0
 
+        if self._sum_episode_return:
```

Comment:
Can we log this by default, as opposed to a flag?

Reply:
Yeah sure

Reply:
Should we remove the flag or keep it with True?